docs/9-membership.md

```diff
@@ -322,6 +190,7 @@ We asked members to share why they think the Protocol Guild is important for Eth
 **Core Contributor** - "I believe this experiment will result in a great precedent on how to keep the core values of the Ethereum protocol alive and relevant." <br><br>
 **Core Contributor** - "I think open source is really important today and is one of the pillars of the world today, and it is going to be more and more important in future, but as today a proper way to reward its developers is missing, so I hope that this experiment will create the basis for a wider approach that could improve the sustainability of open source development." <br><br>
 **Core Contributor** - "Incentive alignment is one of the core issues that impact the long term health of a software/network. I think the Protocol Guild helps solve a part of the incentive alignment problem by allowing members to gain exposure to the network they are developing. I feel like it will go a long way in helping keep and onboard new core developers, as well as ensure everyone has skin in the game." <br><br>
+**Core Contributor** - "it is an interesting concept to fund the Ethereum project, provides both funds and incentives/reward for core devs to do more, and that in my opinion set this open source community a part from the rest" <br><br>
```

Comment:
Might want to capitalized the first word of their quote for consistency.

Reply:
Also updated! Thankee.